GoogleForCreators/web-stories-wp
# Editor: Fixed zoomed scroll in RTL

## Context

This fixed the scroll bug while zoomed in RTL.

NB: This PR is towards #10127 until that gets merged, as it impacts this PR.

## User-facing changes

_Notice the scrollbar being all the way to the left side, but the canvas is only following along correctly in the latter screenshot._

| Before | After |
|-|-|
| ![Screen Shot 2022-01-07 at 16 40 43](https://user-images.githubusercontent.com/637548/148611020-7e3152e7-e262-4277-970e-2c803410c668.png) | ![Screen Shot 2022-01-07 at 16 40 17](https://user-images.githubusercontent.com/637548/148611068-14330de9-dffd-466b-a87c-25cb31604940.png) |

## Testing Instructions

This PR can be tested by following these steps:

1. Switch to RTL
2. Zoom to 200%
3. Scroll horizontally
4. Observe the canvas does not follow while scrolling horizontally (only vertically)

## Checklist




---

Fixes #9807
